cumulus_library/studies/core/core_templates/observation.sql.jinja

```diff
@@ -0,0 +1,72 @@
+{% import 'core_utils.jinja' as utils %}
+
+CREATE TABLE core__observation AS
+WITH temp_observation AS (
+    SELECT
+        {{- utils.basic_cols(
+                'observation',
+                'o',
+                ['id','status', 'effectivedatetime']
```

Comment:
`effectiveDateTime` is an optional field - it should be in a nullable cols call, right?